Muart-C/politico
# #163901278 add authentication token to protect endpoints

**What does this PR do?**
- Add Authorization tokens to access protected endpoints

**Describe the tasks to be completed**
- Add Authorization headers to endpoints.

**How should this  be  manually tested?**
-  `git clone https://github.com/Muart-C/politico.git`

- `cd politico`

- `git checkout bg-add-auth-token-163901278`

-  `$ export FLASK_APP="run.py"`
   `$ export FLASK_ENV="testing"`
   `$ export FLASK_DEBUG=1`
   `$ export DATABASE_POLITICO_DEV="your_test_database_url"`
   `$ export DATABASE_POLITICO_TEST_LOCAL="your_test_database_url`
   `$ pytest -v`
- All tests should passed since authorization headers tokens have been added to the endpoints.
 
**What are the relevant pivotal tracker stories**
https://www.pivotaltracker.com/story/show/163901278
